README.md

```diff
@@ -23,12 +23,14 @@ In case of certificates you need to setup the following directory structure:
       * prod_key.pem - Production APNS app certificate's private key (has to be unencrypted)
       * dev_cert.pem - Development APNS app certificate
       * dev_key.pem - Development APNS app certificate's private key (has to be unencrypted)
+    * fcm/
+      * token.json - Project's FCM token
```

Comment:
Google's calls it FCM's "service account JSON file", so maybe let's try to use their naming across the board.